ffmuc-mesh-vpn-wireguard-vxlan/files/lib/gluon/gluon-mesh-wireguard-vxlan/checkuplink

```diff
@@ -75,25 +86,24 @@ if [ "$(uci get wireguard.mesh_vpn.enabled)" = "true" ] || [ "$(uci get wireguar
 	if [ "$CONNECTED" -ne "1" ]; then
 		logger -t checkuplink "Reconnecting ..."
 		NTP_SERVERS=$(uci get system.ntp.server)
-		# shellcheck disable=SC3060  # busybox sh supports string replacement
-		NTP_SERVERS="${NTP_SERVERS// / -p }"  # each separate NTP server needs to be behind a "-p"
-		# shellcheck disable=SC2086  # we need to expand the list of NTP_SERVERS here
-		if ! gluon-wan /usr/sbin/ntpd -n -N -S /usr/sbin/ntpd-hotplug -p ${NTP_SERVERS} -q
+		NTP_SERVERS_ADDRS=""
+		for NTP_SERVER in $NTP_SERVERS; do
+			ipv6="$(gluon-wan nslookup "$NTP_SERVER" | grep 'Address:\? [0-9]' | grep -E -o '([a-f0-9:]+:+)+[a-f0-9]+')"
+			ipv4="$(gluon-wan nslookup "$NTP_SERVER" | grep 'Address:\? [0-9]' | grep -oE "\b([0-9]{1,3}\.){3}[0-9]{1,3}\b")"
```

Comment:
maybe call gluon-wan once?

```
addresses=$(gluon-wan nslookup "$NTP_SERVER" | grep 'Address:\? [0-9]')
ipv6=$(echo "$addresses" | grep -E -o '([a-f0-9:]+:+)+[a-f0-9]+')
ipv4=$(echo "$addresses" | grep -oE "\b([0-9]{1,3}\.){3}[0-9]{1,3}\b")
```

Reply:
as gluon-wan also was the source of the problems in the past, it seems best to have it separate in case we need to debug this further (i.e. run it with `sh -x checkuplink`).

Reply:
This is from https://github.com/freifunk-gluon/community-packages/pull/92 (I don't know why GitHub stil shows this as change in this PR, the base branch already contains the commit).
Should be fixed separately, not here.

Reply:
@DasSkelett is it possible that your last rebase is not based on the `main` from this repo?